Builds on my machine:
```
nix build '(with import (builtins.fetchTarball { url = "https://github.com/dtzWill/nixpkgs/archive/update/shadowfox-2.0.1.tar.gz"; }) {}; pkgs.shadowfox)'
```
`shadowfox-updater` seems to run:
```
$ ./result/bin/shadowfox-updater --help
Usage of ./result/bin/shadowfox-updater:
  -generate-uuids
    	Wheter to automatically generate UUIDs or not
  -profile-index int
    	Index of profile to use
  -profile-name string
    	Name of profile to use, if not defined or not found will fallback to profile-index
  -set-dark-theme
    	Wheter to automatically set Firefox's dark theme
  -uninstall
    	Wheter to install or uninstall ShadowFox
```
`compiler` doesn't work, should it?
```
$ ./result/bin/compiler --help
panic: exec: "go": executable file not found in $PATH

goroutine 1 [running]:
main.checkErr(...)
	/build/source/compiler/main.go:10
main.main()
	/build/source/compiler/main.go:36 +0x4cf
```

--- a/pkgs/tools/networking/shadowfox/default.nix
+++ b/pkgs/tools/networking/shadowfox/default.nix
@@ -17,6 +17,12 @@ buildGoModule rec {
 
   buildFlags = "--tags release";
 
+  postInstall = ''
+    # Contains compiler package only used by projects CI, and buildGoModule
+    # installs all binaries by default. So we remove this again.
+    rm $out/bin/compiler
+  '';
+
   meta = with stdenv.lib; {
     description = ''
       This project aims at creating a universal dark theme for Firefox while

